css-brochure-form/styles.css

```diff
@@ -0,0 +1,59 @@
+* {
+  font-family: 'Kanit', sans-serif;
+}
+
+div {
+  margin-bottom: 15px;
+}
+
+h1 {
+  color: rgb(238, 116, 111);
+  text-align: center;
+  font-family: sans-serif;
+  font-size: 20px;
+}
+
+select {
+  background-color: rgb(243, 243, 243);
+  border-radius: 5px;
+  padding: 10px 0 10px 10px;
+  width: 100%;
+  border-color: rgb(213, 215, 215);
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-size: 15px;
+}
+
+input {
+  width: 100%;
+  border-radius: 5px;
+  border-color: rgb(213, 215, 215);
+  padding: 10px 0 10px 10px;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-size: 15px;
+  border-style: solid;
+}
```

Comment:
It looks like these two rules are essentially identical, you could add a class to these elements, and just have one rule for that class.